.travis.yml

```diff
@@ -1,15 +1,18 @@
 language: node_js
 node_js: 12
+before_install:
+- npm install -g npm@latest
```

Comment:
Should we be pinning this to a specific version instead?

Reply:
Agreed - we've been doing `npm@latest` everywhere, but I think we should pin it to `npm@6`.  Fixing!